src/_posts/platform/getting-started/2000-01-01-getting-started-with-modsecurity.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Getting started with ModSecurity on Scalingo
+modified_at: 2022-07-25 00:00:00
+tags: tutorial security firewall modsecurity apache nginx waf
+index: 11
+---
+
+ModSecurity is an open-source Web Application Firewall (WAF) that allows users
+to monitor, log and filter HTTP requests. A very common use case is to rely on
+a set of open-source rules provided by [OWASP](https://owasp.org/) (called
+[OWASP ModSecurity Core Rule Set (CRS)](https://owasp.org/www-project-modsecurity-core-rule-set/))
+to protect a web application against generic classes of vulnerabilities and
+Layer 7 attacks such as SQL injections (SQLi), Cross Site Scripting (XSS), ...
+
+Although originally designed for the Apache HTTP Server, it has grown
+compatible with other technologies such as NGINX or Microsoft IIS, becoming a
+*de facto* reference in the open-source ecosystem.
+
+When enabled, ModSecurity analyzes every incoming HTTP request and compares
+the content of the request with a set of patterns. If there is a match, then it
+will simply run some actions, such as blocking the request, logging a message
+or returning a specific HTTP error.
+
+ModSecurity implements its own Domain Specific Language, called *SecLang*,
+making it easy for users to write and share their own rules (combination of
+pattern and actions).
+
+For this first tutorial, we will use NGINX and the CRS provided by OWASP to
+deploy a WAF on Scalingo within minutes.
```

Comment:
No future tense: you have many uses of the future tense in this doc, I would remove them
Nginx is spelled that way throughout the documentation. Can you update all `NGINX` occurrences with `Nginx`?

```suggestion
For this first tutorial, we use Nginx and the CRS provided by OWASP to
deploy a WAF on Scalingo within minutes.
```